src/Microsoft.Dnx.Tooling/Restore/RestoreCommand.cs

```diff
@@ -234,7 +236,7 @@ public RestoreCommand(FrameworkName fallbackFramework)
                 Reports.Information.WriteLine("Updating the invalid lock file with {0}",
                     "dnu restore --lock".Yellow().Bold());
                 useLockFile = false;
-                Lock = true;
+                relock = true;
```

Comment:
Yes, the rest of the changes are following from this, using the local variable instead of the property and adding tests.